### DISCUSSION TITLE
Documentation is incorrect for 2.3.180 with homenets and unable to monitor bond0

### DISCUSSION THREAD

**Discussion by mjbusch81** [Security-Onion-Solutions/securityonion#8988] [Unsupported Versions] [answered]
First of all I am new to Security Onion. I have installed the Standalone version. I have not been able to get the system to monitor bond0. I was able to monitor the same port via Wireshark and see our tap interface.  I have been going over the editing global.sls and $SENSORNAME_$ROLE.sls. 

The problem is in 2.3.180 I am unable to locate /opt/so/saltstack/local/pillar/minions/$SENSORNAME_$ROLE.sls  instead it links me to /home/admin/SecurityOnion/salt
The only extension files I see are schedule.sls and top.sls. I fail to see anything inside the files with sensor:hnsensor. I drilled down into Zeek and Suricata and only find a init.sls but both files are missing sensor:hnsensor. 

Here is the link to the documentation: 

https://docs.securityonion.net/en/2.3/homenet.html

I also included: 

```
sudo so-test
Replay functionality not enabled; attempting to enable now (may require Internet access)...

Pulling so-tcpreplay image
=========================================================================
Starting tcpreplay...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
local:
----------
          ID: so-tcpreplay
    Function: docker_container.running
      Result: True
     Comment: Created container 'so-tcpreplay'
     Started: 23:58:55.967627
    Duration: 1627.712 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      365f94a315aaf953757e18313df97e57d4a15859eea112cb481ee5755fcfb146
              state:
                  ----------
                  new:
                      running
                  old:
                      None

Summary for local
------------
Succeeded: 1 (changed=1)
Failed:    0
------------
Total states run:     1
Total run time:   1.628 s
Replaying PCAP(s) at 10 Mbps on interface bond0...
Actual: 111557 packets (12981286 bytes) sent in 10.38 seconds
Rated: 1249999.6 Bps, 9.99 Mbps, 10742.09 pps
Flows: 4102 flows, 394.99 fps, 2074477 flow packets, 45106 non-flow
Statistics for network device: bond0
        Successful packets:        55304
        Failed packets:            444
        Truncated packets:         0
        Retried packets (ENOBUFS): 0
        Retried packets (EAGAIN):  0
Replay completed. Warnings shown above are typically expected.
```

**Comment by xfaith** [comment, marked as answer]
I am running .180 in standalone and the files are there.
You cant access the files as a normal users though. 
Since its a personal install I use sudo su and then access the files I need.   
There probably is a smarter way.

**Reply by isaacgolding** [reply]
Is this a bare metal install or a VM?

**Reply by mjbusch81** [reply]
It looks like my issue was that our firewall tap isn't compatible. I decided to try a SPAN port on our core switch and now I am seeing traffic. I did a install bare metal and as a VM. We are looking to stick with a VM since we are in a fairly small environment and it will allow for multiple sensors in the future.
